docker/docker_image.go

```diff
@@ -62,7 +65,7 @@ func (i *dockerImage) Signatures() ([][]byte, error) {
 	return i.cachedSignatures, nil
 }
 
-func (i *dockerImage) Inspect() (types.ImageManifest, error) {
+func (i *dockerImage) Inspect() (*types.ImageInspectInfo, error) {
```

Comment:
The original `types.ImageManifest` was an interface. `types.ImageInspectInfo` (formerly `types.DockerImageManifest`) is a struct.